### DISCUSSION TITLE
OOM Killer with docs.js

### DISCUSSION THREAD

**Discussion by Glandos** [js-fns/date-fns#2379] [Q&A] [answered]
I want to contribute to date-fns, but the build.sh goes berserk with an unlimited number of node process when executing docs.js.

Is it a known problem?

> node --version
v14.16.0

**Comment by tan75** [comment, marked as answer]
Hi @Glandos 

thank you for pointing this out. This is a known problem and hopefully soon will be resolved once we fully migrate to TS.
You can omit this step for now as it is usually done during the release process anyway.
We will update the contribution guide shortly.
Thank you again and looking forward to your PR.
